src/renderer/components/NewInstanceModal.ts

```diff
@@ -0,0 +1,135 @@
+import Instance from "../Instance";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Instance' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5163001)